content-src/components/A11yLinkButton/A11yLinkButton.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export function A11yLinkButton(props) {
+  return (
+    <button {...props} className="a11y-link-button">
```

Comment:
Can you put `type="button"` here as well, before the `{...props}` (so it's possible to override?) Actually it would be a good idea to put `className` before so you can override that too